ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -0,0 +1,31 @@
+# global
+import ivy
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_out,
+)
+
+
+@to_ivy_arrays_and_back
+# @from_zero_dim_arrays_to_scalar
+@handle_numpy_out
+def percentile(a,
+               q,
+               /,
+               *,
+               axis=None,
+               out=None,
+               overwrite_input=False,
+               method="linear",
+               keepdims=False,
+               interpolation=None):
+
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    a = ivy.array(a)
+
+    ret = ivy.percentile(a, q, axis=axis, overwrite_input=overwrite_input,
```

Comment:
ivy does not have a percentile function as of now, please either request to add it to the ivy API, or you may implement this frontend function through composition, more information can be found here, thanks!
ref: https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis

Reply:
Hi @juliagsy, I am also working on implementing percentile on ivy function. I have a PR on this: https://github.com/unifyai/ivy/pull/12217

Reply:
Hello! @pradyyadav Thanks for the information! If so, I'll leave that PR to its assignee and come back to this PR a bit later once the function implementation is merged! Please feel free to ping me again once that dependent PR is merged so that I can come back to reviewing this! Thanks a lot!